src/redux/books/apiActions.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { addBook, fetchError } from './books';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps';
+const appId = 'ux6EkcA2ZXp2cRIPwiAL';
+const baseUrl = axios.create({ baseURL: `${url}/${appId}` });
+export const getBookApi = () => (dispatch) => {
+  baseUrl.get('/books').then((res) => {
+    dispatch(addBook(res.data));
+  }).catch(() => {
+    dispatch(fetchError());
+  });
+};
```

Comment:
- [ ] At the moment, when you reload the browser the books are not fetched from the API. Please kindly ensure the fetching of books works as expected.